core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/puppeteer-utils.ts

```diff
@@ -173,6 +173,31 @@ export class BaseUser {
     inputUploadHandle.uploadFile(fileToUpload);
   }
 
+  /**
+   * This function validates whether an anchor tag is correctly linked
+   * to external PDFs or not. Use this particularly when interacting with
+   * buttons associated with external PDF links, because Puppeteer in
+   * headless mode, does not natively support opening of external PDFs.
```

Comment:
...because Puppeteer, in headless-mode, does not natively support the opening of external PDFs.

Reply:
Addressed.